util/src/rlp/untrusted_rlp.rs

```diff
@@ -193,7 +197,7 @@ impl<'a, 'view> View<'a, 'view> for UntrustedRlp<'a> where 'a: 'view {
 	fn size(&self) -> usize {
 		match self.is_data() {
 			// we can safely unwrap (?) cause its data
-			true => BasicDecoder::payload_info(self.bytes).unwrap().value_len,
+			true => BasicDecoder::payload_info(self.bytes).map(|b| b.value_len).unwrap_or(0),
```

Comment:
This just removes a panic, does not really handle any errors properly (that needs some bigger refactor). Will change the comment to reflect that.